frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -113,8 +117,23 @@ export const InfoContent: React.FC<IInfoContent> = ({
             leftSideLabel={'Classification'}
             rightSideItem={propertyInfo?.classification}
           />
+          {!!propertyInfo?.projectNumbers?.length && (
+            <ThreeColumnItem
+              leftSideLabel={
+                propertyInfo.projectNumbers.length > 1 ? 'Project Numbers' : 'Project Number'
```

Comment:
fancy

Reply:
`'Project Number${propertyInfo.projectNumbers.length && 's'}'`  might be fancier

Reply:
Oooh yes I like that lol

Reply:
Gonna steal that and add it in, much cleaner :D

Reply:
"how unreadable can we make this to normal people for literally no reason" lol